geode-core/src/integrationTest/java/org/apache/geode/internal/statistics/DistributedSystemStatisticsIntegrationTest.java

```diff
@@ -187,10 +187,9 @@ public void testAccessingLongStat() {
     Statistics stats = setUpLongStatistics(1);
 
     assertThat(stats.getLong(this.statName1)).isEqualTo(0L);
+    assertThat(stats.getInt(this.statName1)).isEqualTo(0);
```

Comment:
I'm really liking how this was tested!  I don't think it would have ever dawned on me to check both int and long.